src/App/Fossa/ListTargets.hs

```diff
@@ -77,6 +78,9 @@ listTargetsMain ListTargetsConfig{..} = do
     . withTaskPool capabilities updateProgress
     . runAtomicCounter
     . runReader experimental
+    -- The current version of `fossa list-targets` does not support filters.
```

Comment:
Can we include this comment as a warning or info message, so the user is aware that we have not applied any discovery exclusion from the configuration file.